test/e2e/beat/config_test.go

```diff
@@ -259,7 +259,7 @@ func TestPacketbeatConfig(t *testing.T) {
 			beat.HasEvent("event.dataset:dns"),
 		)
 
-	if test.Ctx().Provider != "kind" || test.Ctx().KubernetesVersion != "1.12" {
+	if test.Ctx().Provider != "kind" || test.Ctx().KubernetesMajorMinor() != "1.12" {
```

Comment:
Nit: this is not related to this PR but I was distracted because this condition is written in the other way around in `test/e2e/beat/recipes_test.go`.
```suggestion
	if !(test.Ctx().Provider == "kind" && test.Ctx().KubernetesMajorMinor() == "1.12") {
```